sources/install/package_wordlists.sh

```diff
@@ -49,9 +49,9 @@ function install_seclists() {
 
 function install_pass_station() {
     colorecho "Installing Pass Station"
-    rvm use 3.0.0@pass-station --create
+    rvm use 3.1.2@pass-station --create
     gem install pass-station
-    rvm use 3.0.0@default
+    rvm use 3.1.2@default
```

Comment:
Bad version here

Reply:
pass-station is not compatible with the version 3.2.2